partner_preview_scripts/encore_parser.py

```diff
@@ -514,13 +514,13 @@ def parse_experiment(self, parameters: dict) -> DataFrame:
         return evidence_df
 
 
-def main(outputFile: str, parameters: dict, cellPassportFile: str) -> None:
+def main(outputFile: str, parameters: dict, cellPassportFile: str, cellLineToUberonMapping: str) -> None:
 
     # Initialize spark session
     spark = initialize_sparksession()
 
     # Opening and parsing the cell passport data from Sanger:
-    cell_passport_df = GenerateDiseaseCellLines(cellPassportFile, spark).get_mapping()
+    cell_passport_df = GenerateDiseaseCellLines(spark, cellPassportFile, cellLineToUberonMapping).generate_disease_cell_lines()
```

Comment:
The reason the encore parser is in such a bad shape is that the overall sentiment about encore data itself is extremely bad. Everyone assumes that the data is not good, and new analysis hopefully fixes these issues. As we have been waiting for ~2 years for the new release, and two contact people have left the encore team since, we have a good reason to assume, that the data we'll eventually get would be completely different from what we have now, so the entire parser will likely be rewritten. This project has a history and you are lucky that you are not part of this history.